src/forms/FormField.js

```diff
@@ -115,50 +119,67 @@ export default class FormField extends Component {
   render() {
     return this.props.connectDragSource(
       <div>
+        { this.renderContainer() }
         {
-          !this.props.isDragging && this.state.expanded
+          /*!this.props.isDragging && this.state.expanded
           ? this.renderExpanded()
-          : this.renderCollapsed()
+          : this.renderCollapsed()*/
         }
       </div>
     );
   }
 
-  renderCollapsed() {
+  getIcon(fieldType) {
+    for (var i in askTypes) {
+      if (askTypes[i].friendlyType == fieldType) return askTypes[i].icon;
```

Comment:
I think this could just be `return askTypes.find(type => type.friendlyType === fieldType).icon;`